ci_framework/hooks/playbooks/fetch_compute_facts.yml

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
I got hit by this [error](https://github.com/openstack-k8s-operators/openstack-operator/pull/395#issuecomment-1637849929) Can we add a check here to skip this hook when there is no `compute` in hostvars?

Reply:
that hook should be disabled in case of no compute imho. There's no value to it, if no compute.